source/release_notes/4.0.27-22.rst

```diff
@@ -0,0 +1,19 @@
+.. _PSMDB-4.0.27-22:
+
+================================================================================
+*Percona Server for MongoDB* 4.0.27-22
+================================================================================
+
+:Date: September 28, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.0/install/index.html>`_
+
+Percona Server for MongoDB 4.0.27-22 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for `MongoDB 4.0.27 Community Edition <https://docs.mongodb.com/manual/release-notes/4.0/#4.0.27---sep-13--2021>`_.
+It supports MongoDB 4.0.27 protocols and drivers.
+
+This release is based on MongoDB 4.0.27 and does not include any additional changes.
```

Comment:
I'd like to suggest to make it somewhat more explicit that PSDMB 4.0.27 contains all the changes included in upstream MongoDB 4.0.27:
```suggestion
Percona Server for MongoDB 4.0.27-22 is an enhanced, source available, and highly-scalable database that is a
fully-compatible, drop-in replacement for `MongoDB 4.0.27 Community Edition.
It supports MongoDB 4.0.27 protocols and drivers.

Please see the `MongoDB 4.0.27 Release Notes <https://docs.mongodb.com/manual/release-notes/4.0/#4.0.27---sep-13--2021>`_ for changes made in the upstream release.

In addition to these, Percona Server for MongoDB 4.0.27 includes the following changes:
```

Reply:
We are already saying that with the "fully-compatible drop-in replacement" and providing a direct link to the  MongoDB release notes, aren't we?

Reply:
Besides, we are breaking consistency with other MongoDB versions and Percona Server release notes.

Reply:
We are, but to me linking to the release notes from that section feels less intuitive (I would not expect it linking to the release notes/changelog). Feel free to ignore this recommendation for now, looks like we need to have a more thorough discussion/review of the release note framework elsewhere instead on this PR ;)